week_10/진명재/week10_1043.java

```diff
@@ -0,0 +1,75 @@
+package week_10.진명재;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.*;
+import java.util.stream.IntStream;
+
+public class week10_1043 {
+    static ArrayList<Integer>[] arr;
```

Comment:
```suggestion

    static ArrayList<Integer>[] arr;
    
```
앞 뒤로 띄워서 분류하고 그리고 List를 사용하면 List<List<Integer>> 이렇게 사용하는게 좋음